src/mixins/canvas_events.mixin.js

```diff
@@ -662,6 +694,14 @@
       this._target = this._currentTransform ? this._currentTransform.target : this.findTarget(e) || null;
     },
 
+    /**
+     * @private
+     * @param {Event} event Event object fired on event
+     */
+    _cachePointerId: function(event) {
+      this._pointerId = this._getPointerId(event);
+    },
```

Comment:
let's avoid the caching logic, extracing a value from an event is not worth the extra methods.

Reply:
It is needed to remember the pointerId. I renamed the method to make it more clear.